packages/bootstrap-4/src/UpDownWidget/UpDownWidget.tsx

```diff
@@ -38,7 +38,7 @@ const UpDownWidget = ({
         type="number"
         disabled={disabled}
         readOnly={readonly}
-        value={value ? value : ""}
+        value={value != null ? value : ""}
```

Comment:
Let's do this for clarity and consistency with the other themes (I believe fluent-ui):

```suggestion
        value={value || value === 0 ? value : ""}
```

Reply:
How would you feel about nullish coalescing? `value ?? ''`? I think my tweaks to the babel build should transpile it to something like this.

Reply:
I take that back, not so in the unit tests...